### ISSUE TITLE
Typings for Firebase

### ISSUE THREAD

**Issue by Karasuni** [typings/registry#604]
Similar to #123, since the recent update to Firebase 3.0 the typings [at DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/firebase/firebase.d.ts) are incompatible and outdated. Could someone generate compatible type definitions?

**Comment by unional** [comment]
Do you mean "angularfire"?

**Comment by Karasuni** [comment]
@unional Yes, I'm sorry! It's a bit late and I'm distracted by a build file that doesn't want to work.
